addons/l10n_hr/models/account_move.py

```diff
@@ -0,0 +1,103 @@
+            ('P8', "P8: Issuing invoices with references to the shipping and receipt notes"),
+            ('P9', "P9: Credits or invoices with negative amounts, issued for various reasons, including empty returns packaging"),
+            ('P10', "P10: Issuing a corrective invoice (reversal/correction of invoice)"),
+            ('P11', "P11: Issuing partial and final invoices"),
+            ('P12', "P12: Self-issuance of invoice"),
+            ('P99', "P99: Customer-defined process"),
+        ],
+        string="Business process type",
+        default='P99'
+    )
+    l10n_hr_invoice_time = fields.Datetime(string="Time of invoicing", copy=False)
+    l10n_hr_fiscal_user_id = fields.Many2one(
+        comodel_name="res.partner",
+        string="Fiscal User",
+        domain=lambda self: self._get_l10n_hr_fiskal_user_id_domain(),
+        store=True,
+    )
+    l10n_hr_operator_name = fields.Char(
+        string="Operator Label",
+        depends=['l10n_hr_fiscal_user_id'],
+    )
+    l10n_hr_operator_oib = fields.Char(
+        string="Operator OIB",
+        depends=['l10n_hr_fiscal_user_id'],
+    )
+    l10n_hr_fiscalization_number = fields.Char(
+        string='Invoice fiscalization number',
+        copy=False,
+        readonly=True,
+        compute='_compute_l10n_hr_fiscalization_number',
+    )
+
+    @api.depends('move_type', 'name', 'company_id', 'company_id.country_id')
+    def _compute_l10n_hr_fiscalization_number(self):
+        """
+        Extract the fiscal numbering triple from the document name.
+        Only applies for Croatian sales invoices/credit notes. Expected name
+        pattern is produced by the overridden `sequence.mixin` logic.
+        """
+        for move in self:
+            move.l10n_hr_fiscalization_number = False
+            if (move.move_type in ['out_invoice', 'out_refund']
+                and move.name
+                and move.company_id.country_id.code == 'HR'):
+                parts = move.name.split('-')
+                if len(parts) >= 3:
+                    move.l10n_hr_fiscalization_number = str(int(parts[2].split('/')[0])) + '/' + '/'.join(parts[2].split('/')[1:])
+
+    def _get_l10n_hr_invoice_reference_odoo_invoice(self):
+        """
+        Override to propose a structured reference for HR domestic flows.
+        When the company and partner are both in Croatia and the invoice has a
+        computed fiscalization number, return a reference like:
+          "HR00 {BrOznRacOznPosPrOznNapUr}"
+        where slashes are removed per Croatian banking conventions.
+        """
+        self.ensure_one()
+        # Check if invoice has fiscalization number, company is in Croatia, and partner is in Croatia
+        if (self.l10n_hr_fiscalization_number
+            and self.company_id.country_id.code == 'HR'
+            and self.partner_id.country_id.code == 'HR'):
+            fisc_num_with_dash = self.l10n_hr_fiscalization_number.replace('/', '')
```

Comment:
```suggestion
            fisc_num_with_dash = re.sub(r'\D', '', self.l10n_hr_fiscalization_number)
```
Reference format update: Per Croatian banking conventions, the fiscalization reference should contain only digits. The current implementation using replace('/', '') only removes slashes, but letters and other non-digit characters should also be removed. Using re.sub(r'\D', '', self.l10n_hr_fiscalization_number) correctly extracts only the numeric components.
needed: import re